app/views/collaborations/_form.html.erb

```diff
@@ -13,30 +13,18 @@
   <% end %>
 
   <p>This is the owner:
-<%=  @wiky.user.email %></p>
+  <%=  @wiky.user.email %></p>
 
   <strong>Delete Collaborators</strong><br>
   <table class= "table table-striped">
-    
-
-
-  <% @user_ids_array.each do |user_id| %>
-  
-   
-    <tr>
-     <td><%= User.find(user_id).email %></td> 
-     <td><%= link_to('Destroy', wiky_collaboration_path(@wiky, (
-      @wiky.collaborations.where(user_id: user_id).pluck(:id))), 
-      :method => 'delete', controller: :collaborations)  %></td>
-    </tr>
-   
-
-  <% end %> 
- </table>
-
-
-
-
+    <% @collaborations.reload.each do |collaboration| %>
```

Comment:
@LuisMenjivar 

I'm a little surprised to see `.reload` here, `@collaborations.each do ...` would normally be enough. 

What problem does the `reload` call here solve?